common/static/js/spec_helpers/ajax_helpers.js

```diff
@@ -109,6 +109,10 @@ define(['sinon', 'underscore'], function(sinon, _) {
         );
     };
 
+    respondWithNoContent = function(requests, requestIndex) {
+        respondToDelete(requests, requestIndex);
```

Comment:
In fact, maybe we should just get rid of ``respondToDelete`` since that seems much more specific than everything else in this file. Tests that want to return mock responses to a delete request could just use ``respondWithNoContent``.